competition-simulator.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: page
+title: Competition Simulator
+---
+
+Competition Simulator
+============
+
+For SR2020, a virtual competition will be run using a brand new simulator.
+
+<div class="info">
+  This is different to the <a href="/docs/programming/simulator/">regular simulator</a>.
+</div>
+
+## Installation
+
+### Prerequisites
+
+This simulation is based in [Webots](https://cyberbotics.com/#download), which will need to be downloaded and installed.
+
+You will also need Python installed. The simulator supports >=3.4.
+
+### Installing the simulation
+
+1. [Download the simulation](), and unzip it somewhere on your computer.
```

Comment:
❌ Incomplete link (I realise this is intentional while we haven't done a first release, highlighting so we don't miss this).

Reply:
I've tracked this in the PR description :+1: